source/jormungandr/jormungandr/scenarios/ridesharing/tests/ridesharing_service_manager_test.py

```diff
@@ -158,6 +164,30 @@ def two_same_ridesharing_service_manager_config_from_file_and_db_test():
     assert len(services) == 1
 
 
+def ridesharing_service_manager_config_from_file_and_db_test():
+    """
+    creation of a ridesharing service from database: InstantSystem
+    creation of a ridesharing service from database : Blablacar
+    >> We have to find both services Blablacar and InstantSystem
```

Comment:
out of curiosity, does `>>` means anything special ? (like `>>>` for [Doctest](https://docs.python.org/3/library/doctest.html))

Reply:
:+1: